src/components/Map/Map.js

```diff
@@ -3,7 +3,8 @@ import { bool, number, object, string } from 'prop-types';
 import classNames from 'classnames';
 import { propTypes } from '../../util/types';
 import config from '../../config';
-import { StaticMap, DynamicMap, isMapsLibLoaded } from './GoogleMap';
+// import { StaticMap, DynamicMap, isMapsLibLoaded } from './GoogleMap';
```

Comment:
Is this comment something we'd leave to the code as an example on how to change the map implementation. Or remove when the map migration is complete?

Reply:
Oops, the Mapbox one should be commented out and the Google one in use. I'll fix that.

I'd leave the Mapbox imports as comments so we can easily test those while doing the whole map provider migration. We can decide to drop them when putting it all together in the end.